sdk/python/kfp/components/_data_passing.py

```diff
@@ -139,23 +139,24 @@ def get_canonical_type_for_type_struct(type_struct) -> str:
         return None
 
 
-def get_deserializer_code_for_type(typ) -> str:
-    try:
-        return type_name_to_deserializer.get(get_canonical_type_struct_for_type[typ], None)
-    except:
-        return None
+# Doesn't seem to be used anymore, can be removed?
+# def get_deserializer_code_for_type(typ) -> str:
+#     try:
+#         return type_name_to_deserializer.get(get_canonical_type_struct_for_type[typ], None)
+#     except:
+#         return None
 
 
-def get_deserializer_code_for_type_struct(type_struct) -> str:
+def get_deserializer_code_for_type_struct(type_struct: str) -> str:
```

Comment:
Not entirely sure what the distinction between `type_struct` and `type_name` is, but the value that comes into this method (and `get_serializer_func_for_type_struct`) is effectively the `type_name` (ie. `"Dict[str, str]"`) so wondering if renaming these methods and arguments might for more legible code.

Reply:
It looks to me `type_struct` and `type_name` are the same thing. Please feel free to rename the methods and arguments, and add type annotations and docstrings if you like.